doc/rados/operations/stretch-mode.rst

```diff
@@ -247,6 +247,34 @@ possible, if needed).
 
 .. _Changing Monitor elections: ../change-mon-elections
 
+Exiting Stretch Mode
+=====================
+To exit stretch mode, run the following command:
+
+.. prompt:: bash $
+
+   ceph mon disable_stretch_mode [{crush_rule}]
+
+
+.. describe:: {crush_rule}
+
+   The CRUSH rule that the user wants all the pools to move back to. If this
+   is not specified, the cluster will move back to the default CRUSH rule.
+
+   :Type: String
+   :Required: No.
+
+The command will move the cluster back to normal mode,
+and the cluster will no longer be in stretch mode.
+All pools will move its ``size`` and ``min_size``
+back to the default values it started with.
+At this point the user is responsible for scaling down the cluster
+to the desired number of OSDs.
```

Comment:
Why would exiting stretch mode necessarily mean taking out OSDs?

Reply:
One of the use cases is that the user may encounter a whole DC failure and doesn't want to bother bringing the failed DC back up and wants to continue using the surviving DC without stretch mode.

Reply:
I suspected as much, but as written this might confuse a user who thinks that scaling OSDs is necessary in all cases.

Reply:
@anthonyeleven 
That is a good point, I added.

```
At this point the user is responsible for scaling down the cluster
to the desired number of OSDs if they choose to operate with less number of OSDs.
```

Let me know if you think this clarifies the point I'm trying to make or should I just delete this sentence completely.

Thanks